### PR TITLE
[Model] Add Support for OmChat Model

### PR DESCRIPTION
**Pull request by mary-0830**
This pull request introduces support for the OmChat model.

If you have any questions, please let me know in time!

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,3 +24,6 @@ transformers
 typing_extensions==4.7.1
 validators
 xlsxwriter
+timm
+einops
+flash_attn
```
Comment (kennymckormick) [inline requirements.txt:29]: Should not include those model specific dependencies (a large number of MLLMs do not depend on those libs), which will make it more difficult to install the codebase.

File: vlmeval/api/gpt.py [hunk 2]
```diff
@@ -197,6 +197,8 @@ def generate_inner(self, inputs, **kwargs) -> str:
         ret_code = response.status_code
         ret_code = 0 if (200 <= int(ret_code) < 300) else ret_code
         answer = self.fail_msg
+        import time
+        time.sleep(5)
```
Comment (kennymckormick) [inline vlmeval/api/gpt.py:201]: Do not add the manual sleep. Set a large `retry` can be a better practice.

File: vlmeval/dataset/utils/judge_util.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def build_judge(**kwargs):
             'gpt-4-0125': 'gpt-4-0125-preview',
             'gpt-4-0409': 'gpt-4-turbo-2024-04-09',
             'chatgpt-1106': 'gpt-3.5-turbo-1106',
-            'chatgpt-0125': 'gpt-3.5-turbo-0125',
+            'chatgpt-0125': 'gpt-3.5-turbo', #'gpt-3.5-turbo-0125',
```
Comment (kennymckormick) [inline vlmeval/dataset/utils/judge_util.py:20]: It's preferred to set the specific version clearly, do not use `gpt-3.5-turbo` (we don't know the version).

File: .gitignore [hunk 4]
```diff
@@ -159,3 +159,9 @@ images/
 
 scripts/*ttf
 cache_dir/*
+
+outputs
+omchat*.sh
+omchat_*
+omchat-*
+.vscode/
```
Comment (kennymckormick) [inline .gitignore:167]: Do not add  model-specific entries, you can keep `outputs` and `.vscode`

**Comment by kennymckormick** [comment]
@mary-0830 Please refine the PR according to the comments.  BTW, please run pre-commit before committing the code (more instructions in the Development doc: https://github.com/open-compass/VLMEvalKit/blob/main/docs/en/advanced_guides/Development.md).

**Comment by mary-0830** [comment]
> @mary-0830 Please refine the PR according to the comments. BTW, please run pre-commit before committing the code (more instructions in the Development doc: https://github.com/open-compass/VLMEvalKit/blob/main/docs/en/advanced_guides/Development.md).

I modified the code as required and checked it. Please confirm again, thank you!

**Review by kennymckormick — APPROVED** [review]

**Comment by kennymckormick** [comment]
> @mary-0830 Please refine the PR according to the comments. BTW, please run pre-commit before committing the code (more instructions in the Development doc: https://github.com/open-compass/VLMEvalKit/blob/main/docs/en/advanced_guides/Development.md).

Thanks for the prompt reply.

**Pull request merged**